game/src/ui/gameplay/Arena/tile-selector/TileSelector.gd

```diff
@@ -4,7 +4,7 @@ class_name TileSelector
 # DUPLICATE CODE: must also modify at map
 const TILE_SIZE = 64.0
 enum TILE_CONTENTS { PATH, EMPTY, TOWER, HOME, INVALID = -1 }
-enum ACTION { BUILDING, INSPECTING }
+enum ACTION { BUILDING, INSPECTING, IDLE }
```

Comment:
What is idle mode btw?

Different options:
a. selected a tower in the build mode -> cursor carries tower and will place on the map if valid tile clicked
b. selected a tower in the map -> opens inspector mode
c. ? -> ?

Reply:
i just need idle so that when the game starts and there is no selected tower yet. and u click on a empty tile nth happens

Reply:
got it.